<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ISearchDatabase</class>
 <widget class="QDialog" name="ISearchDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineQuery">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelHelp">
     <property name="text">
      <string>Type the word you want to search for and press the search button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeResult">
     <column>
      <property name="text">
       <string notr="true"/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
